x-pack/plugins/reporting/server/export_types/csv_from_savedobject/create_job.ts

```diff
@@ -22,34 +21,26 @@ import {
 
 export type ImmediateCreateJobFn = (
   jobParams: JobParamsPanelCsv,
-  headers: KibanaRequest['headers'],
   context: RequestHandlerContext,
   req: KibanaRequest
-) => Promise<{
-  type: string;
-  title: string;
-  jobParams: JobParamsPanelCsv;
-}>;
```

Comment:
`jobParams` was unnecessarily nested in an object that is meant for the full params.